.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: ruby
```

Comment:
Add the following underneath to cache dependencies and speed up build

```yml
cache:
  bundler: true
```

https://docs.travis-ci.com/user/caching/